components/03-modules/header/Header.stories.js

```diff
@@ -119,6 +124,15 @@ storiesOf('Modules/Header', module)
         >
           <alpaca-wishlist :products="products" />
         </alpaca-off-canvas-sidebar>
+        <alpaca-off-canvas-sidebar
```

Comment:
`off canvas sidebar` should be narrower. Please add new prop to set `width` o `max-width`. Please check which one is better

Reply:
I think it could be better to add prop (e.g. `compact`) and base on it add appropriate styles in this case b/c padding also is needed to be changed